### PR TITLE
support use property for AttributeDef

### PR DESCRIPTION
**Pull request by Guite**
(slightly refers #67)

### CODE REVIEW

File: src/SchemaReader.php [hunk 1]
```diff
@@ -490,15 +486,11 @@ private function loadGroup(Schema $schema, DOMElement $node): Closure
         $group->setDoc($this->getDocumentation($node));
         $groupOriginal = $group;
 
-        if ($node->hasAttribute('maxOccurs') || $node->hasAttribute('maxOccurs')) {
+        if ($node->hasAttribute('maxOccurs') || $node->hasAttribute('minOccurs')) {
```
Comment (goetas) [inline src/SchemaReader.php:489]: nice catch

**Comment by goetas** [comment]
hmm, i'm confused... is this backward compatible? as example this relies on it https://github.com/goetas-webservices/xsd2php/blob/df731b2e2fabf96fe9d9c4f2b0ccf39152ee48e3/src/Jms/YamlConverter.php#L155

**Comment by Guite** [comment]
Yes, this is a BC break.
Propose to leave that for a while. Starting with PHP 8 support and other changes (e.g. `choice`) support that will break BC as well, so let's maybe do a major bump afterwards...

**Comment by Guite** [comment]
ready for re-review

**Pull request merged**